buildbot/master.cfg

```diff
@@ -590,6 +590,10 @@ portbuilder_factory.useProgress = False
 portbuilder_factory.workdir = '../build'
 logdir = os.path.join(portbuilder_factory.workdir, 'logs')
 
+portbuilder_factory.addStep(steps.SetPropertyFromCommand(
+    command=['port', 'info', '--version', '--line', util.Interpolate('%(prop:portname)s')],
```

Comment:
This potentially doesn't use the correct prefix for port. Maybe it should have been something like
```
util.WithProperties(prefix) + '/bin/port'
```
instead, but I'm not sure whether that works.